auth-web/src/views/auth/home/DecideBusinessView.vue

```diff
@@ -43,11 +42,16 @@ import LearnMoreButton from '@/components/auth/common/LearnMoreButton.vue'
 export default class DecideBusinessView extends Vue {
   private readonly learnMoreUrl = 'https://smallbusinessbc.ca/article/how-to-choose-the-right-business-structure-for-your-' +
     'small-business/'
+  private readonly selectorWizardUrl = `https://entity-selection-dev.apps.silver.devops.gov.bc.ca`
```

Comment:
Is this URL dynamic? If yes, get it from the configuration. If not provide the production URL.

Reply:
doh, an oversight for sure. 

It is indeed dynamic. Will add to the configuration and will also need to be update for environments, the latter i'm not sure i have access to do Post migration.
Will update 👍

Reply:
Updated!

Reply:
Just fixing a test now that we're using the configHelper.

Reply:
Added them to 1password

Reply:
Thank you! Sorry to add work 😁